### PR TITLE
Move Application classes to namespace

### PR DESCRIPTION
**Pull request by laoneo**
Move Application classes to namespace. Additionally it reverts #15081 as the namespace needs to be adopted as well.

### CODE REVIEW

**Comment by wilsonge** [comment]
You need to run composer install --no-dev

**Comment by laoneo** [comment]
Why, it's not related to composer, just fixes #15081.

**Comment by brianteeman** [comment]
dont forget that I really messed up with 15081 and not only did i accidentally merge it I then committed directly as well https://github.com/joomla/joomla-cms/commits/3.8-dev

**Comment by laoneo** [comment]
I guess you guys want to head on with CMS and not Cms so I'm fixing that in this pr as well because at the moment it is completely broken, in every file, classes with Cms are used and not CMS.

**Comment by brianteeman** [comment]
sorry

**Comment by brianteeman** [comment]
I will not touch git after midnight again

**Comment by laoneo** [comment]
No problem...

**Comment by laoneo** [comment]
So all should be fine now, fixed the tests as well.

File: libraries/vendor/composer/autoload_classmap.php [hunk 1]
```diff
@@ -6,14 +6,801 @@
 $baseDir = dirname(dirname($vendorDir));
 
 return array(
+    'Cache_Lite' => $vendorDir . '/pear/cache_lite/Cache/Lite.php',
     'CallbackFilterIterator' => $vendorDir . '/joomla/compat/src/CallbackFilterIterator.php',
     'EasyPeasyICS' => $vendorDir . '/phpmailer/phpmailer/extras/EasyPeasyICS.php',
+    'File_Iterator' => $vendorDir . '/phpunit/php-file-iterator/src/Iterator.php',
```
Comment (wilsonge) [inline libraries/vendor/composer/autoload_classmap.php]: This is still not right

**Comment by laoneo** [comment]
Done. Dunno why that happened.

**Pull request merged**